tests/stdlib/tos.nim

```diff
@@ -177,7 +177,6 @@ when defined(Linux) or defined(osx):
 
 else:
```

Comment:
This needs to be indented one more level to match `when defined(posix):` at L150. Should fix the tests.

Reply:
not applicable anymore IIRC